indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -872,4 +873,58 @@ private void taskComplete(
     // Notify interested parties
     taskRunnerWorkItem.setResult(taskStatus);
   }
+
+  public Collection<ZkWorker> findLazyworkers(Predicate<ZkWorker> isLazyWorker, int excessWorkers)
```

Comment:
any reason we are doing the limiting within this method as opposed to outside of it?